setup/server/osx/mysql.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+brew install mysql
+pip install mysql-python --no-cache-dir
+brew services start mysql
```

Comment:
I dislike that we're separating installation scripts and adding more directories.

This script should be part of the servers, not a in diff directory.